controller/handler/trial.js

```diff
@@ -124,6 +127,10 @@ function trialView (request, reply) {
 
             const endDate = processRules(ruleValues, Date.now());
 
+            if (request.query.requestType === 'ajaxRequest') {
```

Comment:
I can see that.
A view suddenly becoming an API based off an arbitrary variable (to me) seems unexpected.
Having a separate API would follow the [principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).